src/features/custom-labels-accordion/custom-labels-accordion.js

```diff
@@ -2,16 +2,22 @@ import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { FormattedMessage } from 'react-intl';
-import { Accordion, Headline } from '@folio/stripes/components';
+import { get } from 'lodash';
+
+import {
+  Accordion,
+  Headline,
+} from '@folio/stripes/components';
 
-import Toaster from '../../components/toaster';
 import CustomLabelsSection from '../../components/custom-labels-section';
+import Toaster from '../../components/toaster';
 import { getCustomLabels as getCustomLabelsAction } from '../../redux/actions';
 import selectCustomLabels from '../../redux/selectors/select-custom-labels';
 
 class CustomLabelsAccordion extends Component {
   static propTypes = {
     customLabels: PropTypes.object.isRequired,
+    editMode: PropTypes.bool,
     getCustomLabels: PropTypes.func.isRequired,
```

Comment:
Shouldn't this prop be removed? It's not passed to in `resource-edit-custom-title.js`

Reply:
Just realized that the accordion is connected